mix/ContractCallDataEncoder.cpp

```diff
@@ -134,6 +179,11 @@ unsigned ContractCallDataEncoder::encodeSingleItem(QString const& _data, Solidit
 		catch (std::exception const&)
 		{
 			// manage input as a string.
+			QRegExp strExtract("\"(.*)\""); //check if contains both string and hex value, keep the string.
```

Comment:
There should be no need to handle such inputs. User should only enter one representation.